### ISSUE TITLE
When will the space station support language selection

### ISSUE THREAD

**Issue by yunqiang1** [esi/esi-issues#1359]

**Comment by Blacksmoke16** [comment]
> Is there any solution to this?

No. Given that language was explicitly removed via #1235, it's unlikely it'll be added again.

**Comment by yunqiang1** [comment]
Is there any other way?

**Comment by yunqiang1** [comment]
> > 有什么办法解决这个问题吗？
> 
> [不会。鉴于该语言已通过#1235](https://github.com/esi/esi-issues/issues/1235)明确删除，因此不太可能再次添加。

Is there any other way?

> > Is there any solution to this?
> 
> No. Given that language was explicitly removed via #1235, it's unlikely it'll be added again.

Is there any other way?

**Comment by Blacksmoke16** [comment]
> Is there any other way?

No.

**Comment by yunqiang1** [comment]
> > 还有其他办法吗？
> 
> 没有。除了自己翻译什么的。

**Comment by ccp-pinky** [comment]
These days, for routes that support translations, `zh` is working fine. However, for `stations` there is no translation support in ESI: station names are always English, no matter your language selection.

One can how ever use the SDE to find the translated name, as there language support for station names has been added recently.
